server/routes/contacts/create/start/startCreateContactJourneyController.ts

```diff
@@ -3,18 +3,43 @@ import { v4 as uuidv4 } from 'uuid'
 import { Page } from '../../../../services/auditService'
 import { PageHandler } from '../../../../interfaces/pageHandler'
 import CreateContactJourney = journeys.CreateContactJourney
+import ReturnPointType = journeys.ReturnPointType
+import ReturnPoint = journeys.ReturnPoint
 
 export default class StartCreateContactJourneyController implements PageHandler {
   public PAGE_NAME = Page.CREATE_CONTACT_START_PAGE
 
   private MAX_JOURNEYS = 5
 
-  GET = async (req: Request<{ prisonerNumber: string }, unknown, unknown>, res: Response): Promise<void> => {
+  GET = async (
+    req: Request<
+      { prisonerNumber: string },
+      unknown,
+      unknown,
+      {
+        returnJourneyType?: ReturnPointType
+        returnJourneyId?: string
+      }
+    >,
+    res: Response,
+  ): Promise<void> => {
+    const { returnJourneyType, returnJourneyId } = req.query
+    const { prisonerNumber } = req.params
+    let returnPoint: ReturnPoint
+    if (returnJourneyType === 'MANAGE_PRISONER_CONTACTS') {
+      returnPoint = {
+        type: 'MANAGE_PRISONER_CONTACTS',
+        url: `/prisoner/${prisonerNumber}/contacts/list/${returnJourneyId}`,
+      }
+    } else {
+      returnPoint = { type: 'HOME', url: '/' }
```

Comment:
Can you think of a way to clearly explain this process of switching between journeys? I wonder if it needs something in the README to let developers (even us, when we forget in 3 months time) know how this works?

Reply:
Maybe even just a sentence or two in the README references this file / section?